tests/monotouch-test/NaturalLanguage/EmbeddingTest.cs

```diff
@@ -26,6 +26,13 @@ public void Vector ()
 				if (v == NLLanguage.Unevaluated)
 					continue; // this is not a value provided by Apple.
 
+				switch (v){
```

Comment:
nit: should be `switch (v) {` also CheckXcodeVersion (14, 0))`(spacing)
Larger question - are you expecting this to grow beyond the 1 case in the switch? Otherwise why not:
```
if (v == NLLanguage.Kazakh && !TestRuntime.CheckXcodeVersion (14, 0))
    continue;
```

Reply:
@stephen-hawley Good call on the style!
Yes, I believe that anytime a new language is added in the future xcode versions, we will need a new CheckXcodeVersion (X, X) since the test will attempt to call those languages with previous OS versions and fail the test

Reply:
@stephen-hawley Spacing on the switch addressed! I am not sure about the "also CheckXcodeVersion (14, 0))`(spacing)" but let me know if you would like something else there addressed as well!